components/molecules/footer/FooterLocaleSelector.vue

```diff
@@ -2,7 +2,7 @@
   <NuxtSelectNative
     v-model="locale"
     :options="locales"
-    select-class="appearance-none block w-full bg-none dark:bg-transparent light:bg-white border light:border-gray-200 dark:border-secondary-dark rounded-md py-2 pl-3 pr-10 text-base light:text-gray-500 dark:text-secondary-surface focus:outline-none light:focus:ring-black dark:focus:ring-white light:focus:border-gray-400 dark:focus:border-secondary-light sm:text-sm"
+    select-class="appearance-none font-sans block w-full d-body-bg border light:border-gray-200 dark:border-secondary-dark rounded-md py-2 pl-3 pr-10 text-base light:text-gray-500 dark:text-secondary-surface focus:outline-none light:focus:ring-black dark:focus:ring-white light:focus:border-gray-400 dark:focus:border-secondary-light sm:text-sm"
```

Comment:
Can you revert changes ? I'm working on it on another PR ;-)

Reply:
Ok :)